problems/atbash/atbash.java

```diff
@@ -0,0 +1,29 @@
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class AtBash {
+
+    private static char[] letters = IntStream.rangeClosed('a', 'z')
+        .mapToObj(c -> "" + (char) c).collect(Collectors.joining()).toCharArray();
```

Comment:
{'a', ..., 'z'} would be much clearer and i think shorter. Or you could use "abc...z".toCharArray()